e2e/test/scenarios/onboarding/setup/setup.cy.spec.ts

```diff
@@ -369,81 +378,100 @@ describeWithSnowplow("scenarios > setup", () => {
   });
 
   it("should send snowplow events", () => {
-    // 1 - new_instance_created
-    // 2 - pageview
+    let goodEvents = 0;
+
+    goodEvents++; // 1 - new_instance_created
+    goodEvents++; // 2 - pageview
     cy.visit(`/setup`);
 
-    // 3 - setup/step_seen "welcome"
+    goodEvents++; // 3 - setup/step_seen "welcome"
     expectGoodSnowplowEvent({
       event: "step_seen",
       step_number: 0,
       step: "welcome",
     });
     skipWelcomePage();
-    // 4 - setup/step_seen  "language"
+
+    goodEvents++; // 4 - setup/step_seen  "language"
     expectGoodSnowplowEvent({
       event: "step_seen",
       step_number: 1,
       step: "language",
     });
     selectPreferredLanguageAndContinue();
 
-    // 5 - setup/step_seen "user_info"
+    goodEvents++; // 5 - setup/step_seen "user_info"
     expectGoodSnowplowEvent({
       event: "step_seen",
       step_number: 2,
       step: "user_info",
     });
+
     cy.findByTestId("setup-forms").within(() => {
       fillUserAndContinue({
         ...admin,
         company_name: "Epic team",
       });
 
       cy.findByText("What will you use Metabase for?").should("exist");
-      // 6 - setup/step_seen "usage_question"
+      goodEvents++; // 6 - setup/step_seen "usage_question"
       expectGoodSnowplowEvent({
         event: "step_seen",
         step_number: 3,
         step: "usage_question",
       });
       cy.button("Next").click();
 
-      // 7 - setup/usage_reason_selected
+      goodEvents++; // 7 - setup/usage_reason_selected
       expectGoodSnowplowEvent({
         event: "usage_reason_selected",
         usage_reason: "self-service-analytics",
       });
-      // 8 - setup/step_seen "db_connection"
+
+      goodEvents++; // 8 - setup/step_seen "db_connection"
       expectGoodSnowplowEvent({
         event: "step_seen",
         step_number: 4,
         step: "db_connection",
       });
       cy.findByText("I'll add my data later").click();
-      // 9 - setup/add_data_later_clicked
+
+      goodEvents++; // 9/10 - setup/add_data_later_clicked
       expectGoodSnowplowEvent({
         event: "add_data_later_clicked",
       });
-      // 10 - setup/step_seen "data_usage"
+
+      // This step is only visile on EE builds
+      if (isEE) {
+        goodEvents++; // 10/11 - setup/step_seen "commercial_license"
+        expectGoodSnowplowEvent({
+          event: "step_seen",
+          step_number: 5,
+          step: "license_token",
+        });
+
+        cy.button("Next").click();
+      }
+
+      goodEvents++; // 10/11 - setup/step_seen "data_usage"
       expectGoodSnowplowEvent({
         event: "step_seen",
-        step_number: 5,
+        step_number: 5 + (isEE ? 1 : 0),
```

Comment:
I'd do `isEE ? 6 : 5`. It should be easier to understand since we don't need to compute.